app/models/navigation_items.rb

```diff
@@ -45,7 +45,8 @@ def for_provider_account_nav(current_provider_user, current_controller)
 
       items = []
 
-      if current_provider_user.can_manage_organisations? && Provider.with_permissions_visible_to(current_provider_user).exists?
+      if current_provider_user.can_manage_organisations? && \
```

Comment:
How about a 
```suggestion
      if current_provider_user.autorisation.can_manage_at_least_one_organisation? && \
```
in the style of https://github.com/DFE-Digital/apply-for-teacher-training/pull/2543

Reply:
Nice — done!